packages/venia-concept/src/components/Checkout/__tests__/form.spec.js

```diff
@@ -9,9 +10,15 @@ import Section from '../section';
 import Button from 'src/components/Button';
 
 jest.mock('src/classify');
+jest.mock('../addressForm', () => 'AddressForm');
+jest.mock('../paymentsForm', () => 'PaymentsForm');
+jest.mock('../shippingForm', () => 'ShippingForm');
+jest.mock('react', () => {
+    const React = jest.requireActual('react');
+    return Object.assign(React, { useState: jest.fn(React.useState) });
+});
```

Comment:
There are a lot of ways to write tests, so I tend to let most things pass, but I think this is overkill. See the comments below.